build-scripts/Dockerfile.linux-glibc-arm64

```diff
@@ -24,3 +24,4 @@ RUN --mount=type=tmpfs,target=${BUILD_DIR} cp -R /src/. ${BUILD_DIR}/ \
 
 FROM scratch AS export-stage
 COPY --from=build /out/stacks-inspect /out/blockstack-cli /out/clarity-cli /out/stacks-node /
+## comment
```

Comment:
Should this be removed?

Reply:
💯 this was an artifact from my fork testing a non-text update (dockerfile counts as code since it's not .md/.yml)